libusb/libusb.c

```diff
@@ -19,17 +19,40 @@
 #include "stm32.h"
 #include "usb.h"
 #include "usb_cdc.h"
+#include "usb_hid.h"
```

Comment:
If we can wrap these includes using `#if (ENABLE_CDC_COMM == 1)` / `#elif (ENABLE_HID_WEBUSB_COMM == 1)`, then we can be sure that CDC communication will not be impacted which is explicitly used by the bootloader

Reply:
updated